src/app/features/tech-record/components/edit-tech-record-button/edit-tech-record-button.component.spec.ts

```diff
@@ -183,10 +193,10 @@ describe('EditTechRecordButtonComponent', () => {
             fixture.detectChanges();
             fixture.debugElement.query(By.css('#cancel')).nativeElement.click();
 
-            expect(router.navigate).toHaveBeenCalledWith([]);
```

Comment:
Had to change this one sadly, as it was becoming a pain point to refactor while using active route